sos.spec

```diff
@@ -2,21 +2,22 @@ Summary: A set of tools to gather troubleshooting information from a system
 Name: sos
 Version: 4.5.4
 Release: 1%{?dist}
-Group: Applications/System
 Source0: https://github.com/sosreport/sos/archive/%{name}-%{version}.tar.gz
```

Comment:
@TurboTurtle , since we refer to nonexisting URL here, does it make sense to point here to `https://github.com/sosreport/sos/archive/%{version}.tar.gz` and update https://github.com/sosreport/sos/blob/main/.cirrus.yml#L107 accordingly?